vendor/github.com/spaolacci/murmur3/LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright 2013, Sébastien Paolacci.
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
```

Comment:
Do we need a licenses page in the docs to ensure we highlight libraries like this?

Reply:
The library is licensed under BSD-3, and it seems that we have more vendored libraries licensed under it. Do we have a doc page for listing such dependencies?